# ---------------- /PathArray.msg -------------------
# Custom ROS message for a list of path 

std_msgs/Header header
nav_msgs/Path[] paths
